tsl/src/bgw_policy/continuous_aggregate_api.c

```diff
@@ -143,26 +145,31 @@ json_add_dim_interval_value(JsonbParseState *parse_state, const char *json_label
 	}
 }
 
-static void
-check_valid_interval(Oid dim_type, Oid interval_type, const char *str_msg)
+static Datum
+convert_interval_arg(Oid dim_type, Datum interval, Oid *interval_type, const char *str_msg)
```

Comment:
Isn't this conversion useful for other policies as well (e.g., retention)? I figured this would be in time_utils or policy_utils for use by all policies.

Reply:
Yes, I thought maybe to do that in another PR related to the retention policy.